samples/snippets/csharp/VS_Snippets_CLR_Classic/classic TripleDESCryptoServiceProvider Example/CS/source.cs

```diff
@@ -19,8 +19,8 @@ private static void EncryptData(String inName, String outName, byte[] tdesKey, b
      long totlen = fin.Length;    //This is the total length of the input file.
      int len;                     //This is the number of bytes to be written at a time.
 
-     TripleDESCryptoServiceProvider tdes = new TripleDESCryptoServiceProvider();
-     CryptoStream encStream = new CryptoStream(fout, tdes.CreateEncryptor(tdesKey, tdesIV), CryptoStreamMode.Write);
+     AesCryptoServiceProvider aes = new AesCryptoServiceProvider();
+     CryptoStream encStream = new CryptoStream(fout, aes.CreateEncryptor(aesKey, aesIV), CryptoStreamMode.Write);

```

Comment:
The filename in this one suggests it's for TripleDESCryptoServiceProvider specifically.  If that's not true, maybe it needs a rename (and then this should change from AesCryptoServiceProvider to just Aes)

Reply:
This one is for TripleDes - reverting.